protocol/src/main/resources/protocol.xml

```diff
@@ -42,6 +42,9 @@
       <validValue name="DECISION">25</validValue>
       <validValue name="DECISION_REQUIREMENTS">26</validValue>
       <validValue name="DECISION_EVALUATION">27</validValue>
+
+      <!-- Management records / record not related to process automation -->
+      <validValue name="CHECKPOINT">100</validValue>
```

Comment:
:wrench: how about starting from the bottom, so 255 (or 254?). That way we can leave the some more room for the process automation related records.

Reply:
:+1:

Reply:
255 is used as null/unknown value. So will start with 254.